crates/cdk-mintd/Cargo.toml

```diff
@@ -6,9 +6,14 @@ authors = ["CDK Developers"]
 license = "MIT"
 homepage = "https://github.com/cashubtc/cdk"
 repository = "https://github.com/cashubtc/cdk.git"
-rust-version = "1.63.0"                            # MSRV
 description = "CDK mint binary"
 
+[features]
+default = ["management-rpc"]
```

Comment:
Maybe it's better to have this disabled by default?

Keeps the attack surface smaller for the default setup. Mint operators can still enable it if they need it.

Reply:
I think its better to have in the default binary as I think most operator will want it, though it can be disabled at run time if they do not want to so there there is little more expose there from the excess code. The rpc is not started by default at runtime that does default to not start. 

Related if you are worried about a smaller attack surface via excess cdk code mintd should not be used as it brings in both dbs, and all the ln backends and then chooses these at runtime. I don't think this really increase the attack surface as its code that is not run, but does increase the binary.